frontend/integration-tests/tests/kubevirt/utils/utils.ts

```diff
@@ -0,0 +1,205 @@
+}
+
+/**
+   * Search YAML manifest for a given string. Return true if found.
+   * @param     {string}    needle    String to search in YAML.
+   * @param     {string}    name      Name of the resource.
+   * @param     {string}    namespace Namespace of the resource.
+   * @param     {string}    kind      Kind of the resource.
+   * @returns   {boolean}             True if found, false otherwise.
+   */
+export function searchYAML(needle: string, name: string, namespace: string, kind: string): boolean {
+  const result = execSync(`oc get -o yaml -n ${namespace} ${kind} ${name}`).toString();
+  return result.search(needle) >= 0;
+}
+
+export function getResourceObject(name: string, namespace: string, kind: string) {
+  const resourceJson = execSync(`oc get -o json -n ${namespace} ${kind} ${name}`).toString();
+  return JSON.parse(resourceJson);
+}
+
+/**
+   * Search JSON manifest for a given property and value. Return true if found.
+   * @param     {string}    propertyPath    String representing path to object property.
+   * @param     {string}    value           String representing expected value of searched property.
+   * @param     {string}    name            Name of the resource.
+   * @param     {string}    namespace       Namespace of the resource.
+   * @param     {string}    kind            Kind of the resource.
+   * @returns   {boolean}                   True if found, false otherwise.
+   */
+export function searchJSON(propertyPath: string, value: string, name: string, namespace: string, kind: string): boolean {
+  const resource = getResourceObject(name, namespace, kind);
+  const result = _.get(resource, propertyPath, undefined);
+  return result === value;
+}
+
+export const waitForCount = (elementArrayFinder, targetCount) => {
+  return async() => {
+    const count = await elementArrayFinder.count();
+    return count === targetCount;
+  };
+};
+
+export function execCommandFromCli(command: string) {
+  try {
+    execSync(command);
+  } catch (error) {
+    console.error(`Failed to run ${command}:\n${error}`);
+    throw Error(`Failed to run ${command}:\n${error}`);
+  }
+}
+
+export function exposeService(exposeServices: Set<any>) {
+  const srvArray: Array<any> = [...exposeServices];
+  srvArray.forEach(({name, kind, port, targetPort, exposeName, type, namespace}) => {
+    execCommandFromCli(`virtctl expose ${kind} ${name} --port=${port} --target-port=${targetPort} --name=${exposeName} --type=${type} -n ${namespace}`);
+  });
+}
+
+export async function asyncForEach(iterable, callback) {
+  const array = [...iterable];
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
+  }
+}
+
+export const waitForStringInElement = (element: ElementFinder, needle: string) => {
+  return async() => {
+    const content = await element.getText();
+    return content.includes(needle);
+  };
+};
+
+// eslint-disable-next-line eqeqeq
```

Comment:
`!= undefined` is not a good practice. We should not be disabling this linter rule